### PR TITLE
SVCPLAN-2872: Fix ldap_sync to sync email addresses

### PR DESCRIPTION
**Pull request by billglick**
```
Pull changes from https://github.com/jdmaloney/grafana_tools
Re-add pushd to various scripts to fix sourcing config files
Set allow_assign_grafana_admin
```

This is tested on `metrics-test`.

### CODE REVIEW

File: files/root/grafana_tools/backup_and_sync/sync_grafana.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ pushd $(dirname "$0") > /dev/null
 source ./config
 
 ## Get Grafana backing DB type
-db_type=$(grep "^type = " /etc/grafana/grafana.ini | cut -d' ' -f 3)
+db_type=$(grep "^type = " /etc/grafana/grafana.ini | cut -d' '- f 3)
```
Comment (mollycule) [inline files/root/grafana_tools/backup_and_sync/sync_grafana.sh]: looks like somehow a typo got introduced

Comment (billglick) [reply files/root/grafana_tools/backup_and_sync/sync_grafana.sh]: fixed in latest commit

File: files/root/grafana_tools/ldap_sync/sync.sh [hunk 2]
```diff
@@ -17,7 +17,7 @@ do
 	if [ $1 == "--debug" ]; then
 		echo "Validating Group ${g} is still in LDAP"
 	fi
-	if [ -z "$(/usr/bin/ldapsearch -x "(cn=${g})" -b ou=groups,dc=ncsa,dc=illinois,dc=edu | grep uniqueMember)" ]; then
+        if [ -z "$(/usr/bin/ldapsearch -x "(cn=${g})" -b ou=groups,dc=ncsa,dc=illinois,dc=edu | grep uniqueMember)" ]; then
```
Comment (mollycule) [inline files/root/grafana_tools/ldap_sync/sync.sh]: nitpicking here, but i think this should have less leading spaces. 6 spaces less, i think

Comment (billglick) [reply files/root/grafana_tools/ldap_sync/sync.sh]: I think a tab got replaced with spaces or something.

This is another one of those annoying deals where https://github.com/jdmaloney/grafana_tools is outdated. I really don't want to get nitpicky here.

**Comment by mollycule** [comment]
besides, the [commented nitpick](https://github.com/ncsa/puppet-profile_metrics_alerting/pull/6#discussion_r1089487420), this seems good to me!

File: files/root/grafana_tools/ldap_sync/sync.sh (hunk 2 above)
Comment (billglick) [reply files/root/grafana_tools/ldap_sync/sync.sh]: Fixed in latest commit. Reverted back to original tab instead of spaces.

**Pull request merged**